### DISCUSSION TITLE
https://y9san9.me/

### DISCUSSION THREAD

**Discussion by giscus** [y9san9/public-vault#7] [Announcements]
# https://y9san9.me/

Follow through this website to learn something about me.

https://y9san9.me/

**Comment by demndevel** [comment]
impressive

**Comment by sayalovesu** [comment]
nate higgers

**Reply by y9san9** [reply]
No words... Only emotions

**Comment by Kamillaova** [comment]
bober kurwa

**Comment by y9Kap** [comment]
Kotlin president of the world

**Reply by y9san9** [reply]
Based

**Comment by y2k** [comment]
Зумеры придумали хомпейдж?..

**Reply by y9san9** [reply]
Придумали и придумали, че бубнить то

**Comment by otomir23** [comment]
афигенный сайтиш!!

**Comment by deltawenny** [comment]
:white_check_mark: меня попросили оставить комментарий

**Reply by y9san9** [reply]
😁

**Comment by kirillhoe** [comment]
Ахехахкхах эта аватарка из золотых времен молодой России обожаю

**Comment by nnqnn** [comment]
Санчо ты крут!

**Comment by zhelenskiy** [comment]
Make meetacy great again!

**Comment by professorDeveloper** [comment]
hello

**Reply by y9san9** [reply]
Hello

**Comment by ndiuky** [comment]
пожалуй оставлю тут комментарий

**Comment by itzephir** [comment]
site built by compose multiplatform?

**Reply by demndevel** [reply]
thanks god no

**Reply by y9san9** [reply]
It's not 100mb so yeah it's not

**Comment by zhelenskiy** [comment]
What is the state of Meetacy and Seed? I'm really looking forward to using both of them.

**Reply by y9san9** [reply]
I have fully refactored this website. Now all info about my projects and their state is on the [Verbose page](https://y9san9.me/content/verbose). Meetacy and Seed sit on graveyard here. I hope not forever, though.

**Reply by y9san9** [reply]
By the way, did you hear about the new project Friendly? I can dm you the details if I didn't already

**Comment by kotleni** [comment]
Hey. Cool website by the way.
As usual technical and fun.

PS. I will stole your idea about decimal age :3

**Reply by y9san9** [reply]
Thank you for the feedback. The idea was stolen by me, so feel free to continue this stealing loop :)

**Reply by kotleni** [reply]
Bruh, stealing sequence 🧸

**Comment by professorDeveloper** [comment]
idk what put here

**Comment by justarizzz** [comment]
hi can i steal this website's design??
i'll definitely credit you!!!!!!

**Reply by y9san9** [reply]
nah bro no need to mention (if you want you can!!), it's just an open-source solution which is not mine (Quarz)

**Reply by justarizzz** [reply]
ohhh, gotcha
